pkg/types/clusterconfig/cluster_config_gcp.go

```diff
@@ -136,6 +137,20 @@ var UserGCPValidation = &cr.StructValidation{
 				AllowExplicitNull: true,
 			},
 		},
+		{
+			StructField: "AcceleratorsPerInstance",
+			Int64PtrValidation: &cr.Int64PtrValidation{
+				AllowExplicitNull: true,
+			},
+			DefaultDependentFields: []string{"AcceleratorType"},
```

Comment:
off: I take this field `DefaultDependentFields` (alongside `DefaultDependentFieldsFunc`) can be used for `preemptible` and `on_demand_backup` in https://github.com/cortexlabs/cortex/pull/1791.

Reply:
Possibly, we can discuss this live